### PR TITLE
Constantize playback type

### PR DESCRIPTION
**Pull request by leandromoreira**
While I was creating the [dash shaka playback](https://github.com/leandromoreira/clappr-dash-shaka) I got confused with [`getPlaybackType`](https://github.com/clappr/clappr/blob/master/src/base/playback.js#L72) and we even [documented it](http://clappr.github.io/classes/Playback.html#method_getPlaybackType) wrong. 

I'm not totally safe about `src/playbacks/html_img/html_img.js` though, any thoughts on this?

### CODE REVIEW

**Comment by flavioribeiro** [comment]
refs to #304, we should wait for @ronag considerations

File: src/playbacks/html5_audio/html5_audio.js [hunk 1]
```diff
@@ -61,11 +61,11 @@ export default class HTML5Audio extends Playback {
   }
 
   getPlaybackType() {
-    return [0, undefined, Infinity].indexOf(this.el.duration) >= 0 ? 'live' : 'aod'
+    return [0, undefined, Infinity].indexOf(this.el.duration) >= 0 ? Playback.LIVE : Playback.AOD
   }
 
   stalled() {
-    if (this.getPlaybackType() === 'vod' && this.el.readyState < this.el.HAVE_FUTURE_DATA) {
+    if (this.getPlaybackType() === Playback.VOD && this.el.readyState < this.el.HAVE_FUTURE_DATA) {
```
Comment (flavioribeiro) [inline src/playbacks/html5_audio/html5_audio.js]: I'm sure that we're making a mistake here, we should be checking for `AOD`. However, it seems that @dayvson is working on #466 so we should remove the entire html5_audio soon

Comment (dayvson) [inline src/playbacks/html5_audio/html5_audio.js]: yes @flavioribeiro, I am working on that and I will send the pull request shortly

**Comment by leandromoreira** [comment]
is it right to do `git rebase master` from my branch? 
now it seems that I commited 21 files... what is going to happen if we merge this pr?

**Comment by leandromoreira** [comment]
I tend to follow `rebase` flow almost always. :hammer:

**Comment by bernardocamilo** [comment]
Manually merged (4b172f8cf9615c1e49820574f05a2128e272bb94).

**Pull request closed without merging**